### PR TITLE
doc(k8s): removed unnecessary configurations.

### PR DESCRIPTION
**Pull request by gxthrj**
### What this PR does / why we need it:
1.make config.yaml from the latest version 
2.remove some config not necessary

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible?

### CODE REVIEW

File: kubernetes/README.md [hunk 1]
```diff
@@ -21,7 +21,17 @@
 There are some yaml files for deploying apisix in Kubernetes.
 
 ### Prerequisites
-- Install etcd
+- Install etcd , and set env `etcd_url` in config.sh
```
Comment (juzhiyuan) [inline kubernetes/README.md:24]: We could use ``` to include keywords, e.g `etcd`

File: kubernetes/README.md [hunk 2]
```diff
@@ -21,7 +21,17 @@
...
+
+- Run `config.sh` to generate apisix-gw-config-cm.yaml from the latest `config.yaml`
```
Comment (juzhiyuan) [inline kubernetes/README.md:26]: to generate the `apisix-gw-config-cm.yaml`

File: kubernetes/README.md [hunk 3]
```diff
@@ -21,7 +21,17 @@
...
+
+```
```
Comment (juzhiyuan) [inline kubernetes/README.md:28]:
```sh
# if config.sh have no permission to execute, then execute `chmod +x config.sh` to generate apisix-gw-config-cm.yaml
$ sh config.sh
```

File: kubernetes/README.md [hunk 4]
```diff
@@ -56,7 +66,7 @@ or
 $ kubectl create configmap apisix-gw-config.yaml --from-file=../conf/config.yaml
 ```
 
-##### Note: you should modify etcd addr in config file `apisix-gw-config-cm.yaml` or `../conf/config.yaml` first
+##### Note: you should check etcd addr in config file `apisix-gw-config-cm.yaml` or `../conf/config.yaml` first, make sure the etcd addresses are right.
```
Comment (juzhiyuan) [inline kubernetes/README.md:69]: `are correct`.

File: kubernetes/config.sh [hunk 5]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+export etcd_url='http://here_is_you_etcd_IP:2379'
```
Comment (juzhiyuan) [inline kubernetes/config.sh:3]: `http://$ETCD_IP_ADDRESS:2379`

**Comment by membphis** [comment]
@gxthrj please take a look at the output of Github Action:

https://github.com/apache/incubator-apisix/pull/1891/checks?check_run_id=913422137#step:6:61

```
Scan detected 2 error(s) in 2 file(s):
Reading configuration file [.travis/ASF-Release.cfg]...
Scanning files starting at [./]...
Scan detected 2 error(s) in 2 file(s):
  [./kubernetes/config.sh]:
       1: file does not include required license header.
  [./kubernetes/deployment.yaml]:
     113: file does not end with EOL.
```

**Comment by gxthrj** [comment]
> @gxthrj please take a look at the output of Github Action:
> 
> https://github.com/apache/incubator-apisix/pull/1891/checks?check_run_id=913422137#step:6:61
> 
> ```
> Scan detected 2 error(s) in 2 file(s):
> Reading configuration file [.travis/ASF-Release.cfg]...
> Scanning files starting at [./]...
> Scan detected 2 error(s) in 2 file(s):
>   [./kubernetes/config.sh]:
>        1: file does not include required license header.
>   [./kubernetes/deployment.yaml]:
>      113: file does not end with EOL.
> ```

done

**Pull request merged**